governance-charter.md

```diff
@@ -37,8 +37,10 @@ following:
   the project](https://github.com/cncf/toc/blob/main/proposals/sandbox/opentelemetry.adoc)
 - Define, evolve, and defend the **vision, values, mission, and scope** of the
   project - to establish and maintain the soul of OpenTelemetry.
-- Decide how and when **official releases** of OpenTelemetry artifacts are
-  made and what they include.
+- Define and execute the roadmap for OpenTelemetry, as defined in the project
```

Comment:
```suggestion
- Define and prioritize the roadmap for OpenTelemetry, as defined in the project
```

Prioritize might be better than execute, given that the GC generally isn't implementing most features. We're also not people's bosses; the best that we can do is ask that they focus their efforts on certain things.